<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResultWindow</class>
 <widget class="QDialog" name="ResultWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>720</width>
    <height>360</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Wideness</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QHBoxLayout" name="wideLayout">
         <property name="sizeConstraint">
          <enum>QLayout::SetMaximumSize</enum>
         </property>
         <item>
          <widget class="QScrollArea" name="widenessScrollArea">
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>748</width>
              <height>330</height>
             </rect>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_10">
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#00aa00;&quot;&gt;Green&lt;/span&gt; means &lt;span style=&quot; color:#00aa00;&quot;&gt;significancy&lt;/span&gt;; recommend to &lt;span style=&quot; color:#00aa00;&quot;&gt;follow&lt;/span&gt; the result&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#aa0000;&quot;&gt;Red&lt;/span&gt; means &lt;span style=&quot; color:#aa0000;&quot;&gt;insignificancy&lt;/span&gt;; recommend to &lt;span style=&quot; color:#aa0000;&quot;&gt;disregard&lt;/span&gt; the result&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="skewednessLayout">
               <item>
                <widget class="QLabel" name="skewednessTitle">
                 <property name="text">
                  <string>&lt;b&gt;Skewedness&lt;/b&gt;</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="skewednessUpward">
                 <property name="text">
                  <string>TextLabel</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="skewednessRightward">
                 <property name="text">
                  <string>TextLabel</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QFrame" name="frame">
               <property name="frameShape">
                <enum>QFrame::HLine</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <property name="lineWidth">
                <number>4</number>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QVBoxLayout" name="widenessLayout">
               <item>
                <widget class="QLabel" name="widenessTitle">
                 <property name="text">
                  <string>&lt;b&gt;Wideness&lt;/b&gt;</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="widenessHorizontal">
                 <property name="text">
                  <string>TextLabel</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="widenessVertical">
                 <property name="text">
                  <string>TextLabel</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QFrame" name="frame_2">
               <property name="frameShape">
                <enum>QFrame::HLine</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <property name="lineWidth">
                <number>4</number>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QVBoxLayout" name="widenessRawResultsLayout">
               <item>
                <widget class="QLabel" name="widenessRawResultsTitle">
                 <property name="text">
                  <string>&lt;b&gt;Raw Results&lt;/b&gt;</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QTextEdit" name="widenessRawResults"/>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="wideGraphLayout"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <attribute name="title">
       <string>Rotatedness</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <layout class="QHBoxLayout" name="rotateLayout">
         <property name="sizeConstraint">
          <enum>QLayout::SetMaximumSize</enum>
         </property>
         <item>
          <widget class="QScrollArea" name="rotatednessScrollArea">
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents_2">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>748</width>
              <height>330</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_12">
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#00aa00;&quot;&gt;Green&lt;/span&gt; means &lt;span style=&quot; color:#00aa00;&quot;&gt;significancy&lt;/span&gt;; recommend to &lt;span style=&quot; color:#00aa00;&quot;&gt;follow&lt;/span&gt; the result&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#aa0000;&quot;&gt;Red&lt;/span&gt; means &lt;span style=&quot; color:#aa0000;&quot;&gt;insignificancy&lt;/span&gt;; recommend to &lt;span style=&quot; color:#aa0000;&quot;&gt;disregard&lt;/span&gt; the result&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout">
               <property name="sizeConstraint">
                <enum>QLayout::SetMaximumSize</enum>
               </property>
               <item>
                <widget class="QLabel" name="rotatednessTitle">
                 <property name="text">
                  <string>&lt;b&gt;Rotatedness&lt;/b&gt;</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="rotatednessResult">
                 <property name="text">
                  <string>TextLabel</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QFrame" name="frame_3">
               <property name="frameShape">
                <enum>QFrame::HLine</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <property name="lineWidth">
                <number>4</number>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <item>
                <widget class="QLabel" name="rotatednessRawResultsTitle">
                 <property name="text">
                  <string>&lt;b&gt;Raw Results&lt;/b&gt;</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QTextEdit" name="rotatednessRawResults"/>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="rotateGraphLayout"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <attribute name="title">
       <string>Raw</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QTextEdit" name="rawText"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
